module/zfs/dsl_pool.c

```diff
@@ -347,7 +347,8 @@ dsl_pool_close(dsl_pool_t *dp)
 }
 
 dsl_pool_t *
-dsl_pool_create(spa_t *spa, nvlist_t *zplprops, uint64_t txg)
+dsl_pool_create(spa_t *spa, nvlist_t *zplprops, dsl_crypto_params_t *dcp,
+    uint64_t txg)
```

Comment:
These are specified just as any other filesystem property at pool creation time, with `-O`. I didn't document this because it works the same way as compression, dedup, checksum, etc. Would you like me to add a line somewhere?